### PR TITLE
docs: various updates to match things in 2021

### PR DESCRIPTION
**Pull request by lidel**
This PR  updates  https://dnslink.io:

- make `_dnslink` prefix the rule of thumb  (to follow  https://github.com/ipfs/ipfs-docs/pull/658)
- mention ENS gateway at https://eth.link
- fix links to point at correct places in ipfs-docs v2 site

# TODO
- [x] move this repo to `ipfs/` org
- [x] add CI setup to publish the website like other ones (cid.ipfs.io, docs.ipfs.io etc)

### CODE REVIEW

**Comment by lidel** [comment]
@mburns this recently got moved from Juan's personal account to `ipfs/` org and I'm trying to clean stuff up.

Question: if I want to publish this repo via CI  is it better to use Fleek  (like https://cid.ipfs.io) or our own cluster (like https://docs.ipfs.io)? Any concerns about either or should I do a coin flip?

**Comment by mburns** [comment]
Either would work fine. Generally, I'd say Fleek for sites that are more frequently updated, and Collab-cluster for rarely updated ones.

`dnslink.dev` is already setup on the collab-cluster, so adding CI/CD steps should be trivial.

**Review by lidel — COMMENTED** [review]
I've added GithubAction for building the static website, should work after we merge this to main branch (this PR is from a fork, so we dont see preview just yet).

Need some help with secrets (details inline).

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+      # pin the built site to ipfs-websites cluster, output the cid as `steps.ipfs.outputs.cid`
+      # see: https://github.com/ipfs-shipyard/ipfs-github-action
+      - uses: ipfs-shipyard/ipfs-github-action@v2.0.0
+        id: ipfs
+        with:
+          path_to_add: build
+          cluster_host: /dnsaddr/ipfs-websites.collab.ipfscluster.io
+          cluster_user: ${{ secrets.CLUSTER_USER }}
+          cluster_password: ${{ secrets.CLUSTER_PASSWORD }}
```
[the remark this answers is not in the document]
Comment (mburns) [reply .github/workflows/ci.yml:54]: turns out, I don't have admin(?) access to that on this repo: https://github.com/ipfs/dnslink.dev/settings/secrets/actions

Need to add:
* `CLUSTER_USER`
* `CLUSTER_PASSWORD`
* `GITHUB_TOKEN`
* `DNSIMPLE_TOKEN`

The values are in 1pass, can coordinate on Slack.

[the remark this answers is not in the document]
Comment (lidel) [reply .github/workflows/ci.yml:54]: @mburns I dont have access to those things in 1pass, but you should have admin now (permissions got wonky after move, fixed them). Mind adding those secrets?

[the remark this answers is not in the document]
Comment (mburns) [reply .github/workflows/ci.yml:54]: looks like there is still a permissions issue, I don't have that tab available to me.

I've gotten in touch with andyschwab and he is getting them added.

**Pull request merged**